docs/data-sources.md

```diff
@@ -9,9 +9,29 @@ Mapzen Terrain Tiles are powered by several major open data sets and we owe a tr
 The underlying data sources are a mix of:
 
 - [3DEP](http://nationalmap.gov/elevation.html) (formerly NED) in the United States, 10 meters outside of Alaska, 3 meter in select land and territorial water areas
-- [SRTM](https://lta.cr.usgs.gov/SRTM) globally except high latitudes, 30 meters in land areas
-- [GMTED](http://topotools.cr.usgs.gov/gmted_viewer/) globally, coarser resolutions at 7.5", 15", and 30" in land areas
+- [ArcticDEM](http://nga.maps.arcgis.com/apps/MapSeries/index.html?appid=cf2fba21df7540fb981f8836f2a97e25) strips of 5 meter mosaics across the majority of the land north of 60° latitude
```

Comment:
Nice graphic for posterity [from here](https://www.pgc.umn.edu/data/arcticdem/): 

![image](https://user-images.githubusercontent.com/261584/28716687-880ac6a6-736c-11e7-9e71-e4ad9fde6338.png)